tests/infra/server.py

```diff
@@ -74,7 +74,7 @@ def _inject_api_requests_methods(cls):
     def shutdown(self):
         if self._server is not None:
             self._server.server.shutdown()
-        self.join(timeout=30)
+        self.join(timeout=500)
```

Comment:
What's the motivation for increasing this timeout?

Reply:
Was left in by mistake for more debugging time, reverted.